neuralmonkey/train.py

```diff
@@ -36,9 +37,12 @@ def create_config() -> Configuration:
     config.add_argument("evaluation")
     config.add_argument("runners")
     config.add_argument("test_datasets", required=False, default=[])
-    config.add_argument("logging_period", cond=lambda x: x > 0,
+    # a little "hackish" checking
```

Comment:
nadepiš to "HACK", on to github umí rozpoznávat a pylint taky, když to zapneš, takže pak to jde vygrepovat a najít